src/components/form/HdTextarea.vue

```diff
@@ -11,6 +11,9 @@
     :disabled="disabled"
     @clear-click="clearTextarea"
     @status-click="focusTextarea"
+    :class="{
+      'field--maxlength-indicator-visible': Boolean(maxlength)
```

Comment:
Here you're using `Boolean(maxlength)` and later you use `v-if="maxlength"`, they are both the same, you can just stick to one to be consistent.

Reply:
nice catch, i started to use `isMaxlengthIndicatorVisible` variable instead